tests/test_sticker.py

```diff
@@ -325,6 +325,24 @@ async def test_error_without_required_args(self, bot, chat_id):
         with pytest.raises(TypeError):
             await bot.send_sticker(chat_id)
 
+    @flaky(3, 1)
+    async def test_premium_animation(self, bot):
+        # testing animation sucks a bit since we can't create a premium sticker. What we can do is
+        # get a sticker set which includes a premium sticker and check that specific one.
+        premium_sticker_chat = await bot.get_sticker_set("Flame")
```

Comment:
`_set` instead of `_chat`?

Reply:
yes, you don't get a sticker_chat, you get a set. https://docs.python-telegram-bot.org/en/v20.0a1/telegram.bot.html#telegram.Bot.get_sticker_set ;P

Reply:
That's my point: `premium_sticker_chat` -> `premium_sticker_set` ;P

Reply:
Ohh you meant the variable I only saw the request